metadata/pkg/drivers/aws/aws.go

```diff
@@ -69,28 +70,58 @@ func ObjectList(sess *session.Session, bucket *model.MetaBucket) error {
 		totSize += obj.Size
 		obj.StorageClass = *object.StorageClass
 
+		tags, err := svc.GetObjectTagging(&s3.GetObjectTaggingInput{Bucket: &bucket.Name, Key: &obj.ObjectName})
+
+		if err == nil {
+			tagset := make(map[string]string)
```

Comment:
make may not required.

Reply:
make is being used for map initialisation.